### ISSUE TITLE
Please support Dart 2.13

### ISSUE THREAD

**Issue by ened** [dart-lang/dart-pad#1923]
It would be great to support Dart 2.13, so that the [new features announced](https://medium.com/dartlang/announcing-dart-2-13-c6d547b57067) (type aliases for example) can be tested.

**Comment by domesticmouse** [comment]
@srawlins and @RedBrogdon do you see any problem with lifting the min sdk to 2.13 to enable typedefs?

**Comment by domesticmouse** [comment]
Actually, looking at the code, we already support Dart 2.13. However, it is only available on the Null Safety version of Dart Pad, because the null unsafe version of DartPad is using a min sdk of Dart 2.10 to disable null safety.

Try the following code in the null safe DartPad:

```dart
typedef Integer = int;
void main() {
  print(int == Integer); // prints true in the DartPad console
}
```

**Comment by ened** [comment]
@domesticmouse ok cool. Ill switch to null-safety and try again.

I didn't think of the null-safety switch because the bottom right of the screen shows Dart 2.13 and the error message when using type aliases is about pubspec.yaml etc.

<img width="1001" alt="Screen Shot 2021-06-16 at 22 45 17" src="https://user-images.githubusercontent.com/269860/122298329-872ac000-cef4-11eb-878e-f146cc9741c6.png">

**Comment by RedBrogdon** [comment]
> Actually, looking at the code, we already support Dart 2.13.

I saw the notification for this issue and thought I'd experienced a glitch in the matrix. 😄 

IIRC, we upped the version to 2.13 for the new typedefs that out in that version. 

@ened, you raise a valid point about the version numbers. We're going to do away with the null safety switch and go to opted-in-all-the-time soon, though, at which point it shouldn't be an issue any more.
